### DISCUSSION TITLE
rke2 flannel and Checksum Off

### DISCUSSION THREAD

**Discussion by uk1988** [rancher/rke2#6898] [Q&A] [answered]
Hej All

Like many others, we've encountered a challenge with Kubernetes cross-cluster communication in certain installations. After a week of troubleshooting, we discovered that the root cause is a well-documented issue related to VMware's handling of checksum calculations, particularly when using Flannel with VXLAN as the backend type.

To overcome this hurdle, we found two effective solutions:
1. Switching the backend type for Flannel to host-gw.
2. Disabling checksum on the Flannel interface directly.

Our team has been exploring ways to disable the checksum directly through a Helm chart, but so far, we haven't found a direct method to achieve this. This leads us to question whether Flannel supports the ability to disable checksums for its interface through configuration, or if using ethtool is the only viable approach.

Additionally, it might be beneficial for the RKE2 known issues documentation to highlight that Flannel encounters similar checksum-related issues as Calico. This could help others facing the same problem to identify and implement a solution more swiftly.

I'm reaching out to see if anyone has managed to disable checksums through Flannel's configuration directly or has any insights on this matter. Any shared experiences or advice would be greatly appreciated.

**Comment by manuelbuil** [comment, marked as answer]
Unfortunately, I don't think Flannel supports that option, although it could be a good feature @rbrtbnfgl

**Reply by uk1988** [reply]
thank you for another fast replay, I guess then we will add disabling checksum into our deployment scripts
